Backoffice/Form/Type/SiteType.php

```diff
@@ -65,46 +58,45 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'data-prototype-label-remove' => $this->translator->trans('open_orchestra_backoffice.form.field_option.delete'),
                 ),
                 'options' => array( 'label' => false ),
+                'group_rank' => 4,
             ))
             ->add('blocks', 'oo_block_choice', array(
                 'multiple' => true,
-                'label' => 'open_orchestra_backoffice.form.website.blocks',
-                'required' => false
+                'expanded' => true,
+                'label' => false,
+                'required' => false,
+                'group_rank' => 3,
+                'sub_group' => 'open_orchestra_backoffice.form.website.sub_group.block',
```

Comment:
@itkg-nanne : can you do the same process on sub_group that is done on group, ie introducing a sub_group_rank ?